pkg/boilerplate/operator/option.go

```diff
@@ -6,27 +6,19 @@ import (
 	"github.com/openshift/cluster-authentication-operator/pkg/boilerplate/controller"
 )
 
-// key is the singleton key shared by all events
-// the value is irrelevant, but pandas are awesome
-const key = "🐼"
-
 type Option func(*operator)
 
 func WithInformer(getter controller.InformerGetter, filter controller.Filter, opts ...controller.InformerOption) Option {
-	return toAppendOpt(
```

Comment:
/hold

Restore this function.  This repo has the canonical copy.

Reply:
We can't use `o.name` with this function (service-ca-operator didn't use this function so the change worked). I'd rather remove the unnecessary indirection anyways.

Reply:
blah, sigh.  Shows we are leaking between the two use cases.  oh well.  I had added new options in some unmerged PRs but I guess I will have to deal with the extra boilerplate (or conditionally use `toAppendOpt`).

/hold cancel
/lgtm